MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino

```diff
@@ -0,0 +1,123 @@
+/*
+  KelvinVR Library - Peltier Control Example
+  This sketch demonstrates the use of the cooling and heating methods
+  to control a Peltier module using the L293D driver.
+
+      Arduino Mega 2560
+    ___________________
+   |  ___________  USB |
+   | |           |     |
+   | |           |     |
+   | |           |     |
+   | |           |     |
+   | |___________|     |
+   |                   |
+   |                   |
+   |                   |
+   |                   |
+   |                   |
+   |                   |
+   |                   |
+ --+---+---+---+---+---+---+---+---+---+---+---+
+   | D | D | D | D | D | D | D | . | . | . | D |
+   | 5 | 6 | 7 | 8 | 9 | 1 | 1 |   |   |   | 3 |
+   |   |   |   |   |   | 0 | 1 |   |   |   | 7 |
+ --+---+---+---+---+---+---+---+---+---+---+---+
+     |   |   |   |   |   |   |               |
+     |   |   |   |   |   |   |               |
+     |   |   |   |   |   |   |               |
+     |   |   |   |   |   |   |               Buzzer
+     |   |   |   |   |   |   |
+     |   |   |   |   |   |   +-- Red LED
+     |   |   |   |   |   |
+     |   |   |   |   |   +-- Green LED
+     |   |   |   |   |
+     |   |   |   |   +-- LED+ (Fading LED)
+     |   |   |   |
+     |   |   |   +-- LED-
+     |   |   |
+     |   |   +-- L293D Enable (Peltier)
+     |   |
+     |   +-- L293D Input 1 (Peltier)
+     |
+     +-- L293D Input 2 (Peltier)
+
+Note: Pins 5, 6, 9, 10, and 11 support PWM for LED brightness and motor control
+
+*/
+
+#include <KelvinVR.h>
+
+Kelvin kelvin("Mk1-Alpha");
+
+const int coolingDuration = 10000; // 10 seconds
+const int heatingDuration = 10000; // 10 seconds
+const int idleDuration = 5000;     // 5 seconds
+
+void setup()
+{
+    Serial.begin(9600);
+    while (!Serial)
+    {
+        ; // wait for serial port to connect. Needed for native USB port only
+    }
+
+    kelvin.fullPinSetup();
+
+    Serial.println("KelvinVR Peltier Control Example");
+    Serial.println("================================");
+}
+
+void loop()
+{
+    // Cooling cycle
+    Serial.println("Starting cooling cycle...");
+    kelvin.startCooling(200); // Start cooling at about 78% intensity
```

Comment:
Create additional method for Cooling cycle (Cooling down, blue LED ON, while cycle working fade effect running)

Reply:
Done! This method is called void runThermalCycle(bool isCooling, int intensity, int durationSeconds) :
`    // Run a cooling cycle at 78% intensity for 30 seconds
    kelvin.runThermalCycle(true, 200, 30);`